e2e-tests/portal/cypress/e2e/journeys/dashboard/fixtures/index.js

```diff
@@ -73,6 +74,18 @@ const BSS_FACILITY_BOND_UNISSUED = {
   hasBeenIssued: false,
 };
 
+const BSS_FACILITY_BOND_ISSUED_RISK_EXPIRED = {
+  type: CONSTANTS.FACILITY.FACILITY_TYPE.BOND,
```

Comment:
pedantic but could we dereference up to FACILITY_TYPE? const { FACILITY_TYPE } = CONSTANTS.FACILITY